src/components/organisms/Carousel/Carousel.jsx

```diff
@@ -44,61 +44,43 @@ const Carousel = (props) => {
         <div className="carousel__slider__image-slider">
           <TransitionGroup childFactory={cloneImage(imageDirection)}>
             <CSSTransition
-              key={carousel[activeIndex].path}
+              key={CarouselSlider[activeIndex].path}
               timeout={1000}
               classNames={imageDirection}
             >
-              <img src={carousel[activeIndex].path} alt="carousel" />
+              <img src={CarouselSlider[activeIndex].path} alt={CarouselSlider[activeIndex].altTag}/>
             </CSSTransition>
           </TransitionGroup>
         </div>
         <div className="carousel__slider__text-slider">
           <TransitionGroup childFactory={cloneText(textDirection)}>
             <CSSTransition
```

Comment:
What is this CSStransition?

Reply:
This will be changed in different PR for Carousel